pkg/cmd/repo/create/http.go

```diff
@@ -140,15 +141,16 @@ func repoCreate(client *http.Client, hostname string, input repoCreateInput) (*a
 			return nil, err
 		}
 
-		if !input.HasWikiEnabled || !input.HasIssuesEnabled {
-			updateVariables := map[string]interface{}{
-				"input": updateRepositoryInput{
-					RepositoryID:     response.CloneTemplateRepository.Repository.ID,
-					HasWikiEnabled:   input.HasWikiEnabled,
-					HasIssuesEnabled: input.HasIssuesEnabled,
-				},
-			}
+		updateVariables := map[string]interface{}{
+			"input": updateRepositoryInput{
+				RepositoryID:     response.CloneTemplateRepository.Repository.ID,
+				HasWikiEnabled:   input.HasWikiEnabled,
+				HasIssuesEnabled: input.HasIssuesEnabled,
+				HomepageURL:      input.HomepageURL,
+			},
+		}
```

Comment:
Can we move the construction of `updateVariables` inside the block below, otherwise it is being created for no reason.

Reply:
This is updated now in cc223a9! 👍🏼